packages/wp-story-editor/src/components/documentPane/status/test/status.js

```diff
@@ -86,42 +95,71 @@ describe('statusPanel', () => {
 
   it('should render Status Panel', () => {
     arrange();
-    const element = screen.getByRole('button', {
-      name: 'Visibility',
-    });
-    expect(element).toBeInTheDocument();
+    clickDropdown();
 
-    const radioOptions = screen.getAllByRole('radio');
-    expect(radioOptions).toHaveLength(3);
-    expect(screen.getByLabelText('Public')).toBeInTheDocument();
-    expect(screen.getByLabelText('Private')).toBeInTheDocument();
-    expect(screen.getByLabelText('Password Protected')).toBeInTheDocument();
+    expect(screen.getAllByRole('option')).toHaveLength(3);
+    expect(
+      screen.getByRole('option', {
+        name: 'Selected Public',
+      })
+    ).toBeInTheDocument();
+    expect(
+      screen.getByRole('option', {
+        name: 'Private',
+      })
+    ).toBeInTheDocument();
+    expect(
+      screen.getByRole('option', {
+        name: 'Password Protected',
+      })
+    ).toBeInTheDocument();
   });
 
   it('should always render the "Public" visibility option', () => {
     arrange({
       publish: false,
     });
-    expect(screen.getByLabelText('Public')).toBeInTheDocument();
+    clickDropdown();
+
+    expect(
+      screen.getByRole('option', {
+        name: 'Selected Public',
```

Comment:
Just `Public` is what's getting looked for. 
<img width="1133" alt="Screen Shot 2022-02-17 at 3 25 36 PM" src="https://user-images.githubusercontent.com/10720454/154581551-41996cef-0914-4e7a-addf-4bc3e8d8eff6.png">

Reply:
I am a ding dong.

Reply:
There are no options when it is disabled  🤦🏻‍♀️